### ISSUE TITLE
Missing alt / title parameters from Gcconnex / leaf image tag

### ISSUE THREAD

**Issue by Phanoix** [gctools-outilsgc/gcconnex#11]
Missing alt / title parameters from Gcconnex / leaf image tag.
==> Add description: 'Gcconnex',   also needs closing /
